### ISSUE TITLE
ARM simulator & IDE

### ISSUE THREAD

**Issue by sayan801** [sayan801/ShabdoKhojEngine#5]
https://www.keil.com/demo/eval/arm.htm#/DOWNLOAD

**Comment by suvra-technicise** [comment]
I can't download it and the previous 1 also . The website shows homepage only

**Comment by sayan801** [comment]
no need to download.. We have downloaded already..

Regards,
Chandra Shekhar Sengupta

**Comment by suvra-technicise** [comment]
Yes , but the download  link is not opening...showing homepage of the website only.....So can't have it.
